Bottom: 29b9e08fe9d44d2f701a118b8342fe31b545f0f1
Top:    4e6aa4a637717438e96073db4601629e7139185a
Author: Ian Kent <raven@themaw.net>
Date:   2013-08-19 08:44:15 +0800

Refresh of autofs-5.0.7-fix-bad-mkdir-permission-on-create.patch

---

diff --git a/CHANGELOG b/CHANGELOG
index 2499f1c..3a3fec1 100644
--- a/CHANGELOG
+++ b/CHANGELOG
@@ -75,6 +75,7 @@
 - fix fix wildcard multi map regression.
 - improve timeout option description.
 - only probe specific nfs version if requested.
+- fix bad mkdir permission on create.
 
 25/07/2012 autofs-5.0.7
 =======================
